.github/workflows/build.yml

```diff
@@ -145,34 +141,65 @@ jobs:
         if: github.ref != 'refs/heads/master' && github.event_name == 'pull_request'
         run: |
           [ ! -f changes.md ] || gh pr comment ${PR_NUMBER} --body-file changes.md
-          
-      # - name: Save snapshots
-      #   uses: getsentry/action-visual-snapshot@v2
-      #   with:
-      #     save-only: true
-      #     snapshot-path: .artifacts/visual-snapshots
         
-  # visual-diff:
-  #   permissions:
-  #     id-token: write
-  #     contents: write
-  #     checks: write
-  #   needs: [build]
-  #   name: validate screenshots
-  #   if: github.ref != 'refs/heads/master' && github.event_name == 'pull_request'
-  #   runs-on: ubuntu-latest
-  #   timeout-minutes: 20
-  #   steps:
-  #     - name: AWS Configure
-  #       uses: aws-actions/configure-aws-credentials@v1.6.1
-  #       with:
-  #         aws-region: ap-southeast-2
-  #         mask-aws-account-id: true
-  #         role-to-assume: ${{ secrets.AWS_ROLE_SCREENSHOT }}
-
-  #     - name: Diff snapshots
-  #       id: visual-snapshots-diff
-  #       uses: blacha/action-visual-snapshot@v2
-  #       with:
-  #         storage-prefix: "s3://linz-basemaps-screenshot"
-  #         storage-url: "https://d25mfjh9syaxsr.cloudfront.net"
\ No newline at end of file
+  screenshot:
+    permissions:
+      id-token: write
+      contents: read
+    needs: [build]
+    name: taking screenshots
+    if: github.ref != 'refs/heads/master' && github.event_name == 'pull_request'
```

Comment:
Screenshots have to run on master otherwise there is nothing to compare to.

Reply:
I wonder why this still nothing to compare event I remove the condition of master. Is it because I change the job name as screenshot which previous screenshots are saving in the builds?

Reply:
I am not 100% on this, but the basic idea is the GHA will download artifacts from master's last build and then use those to compare  against the artifacts created by this GHA.

The name of the GHAs may have changed which means it cant find imagery? I dont know though.

Reply:
My thought is because the name changes from `build/build` to `build/taking screenshots`, so there is no imagery in the new location to compare? Might have something to compare once this merged for next run.